src/components/HourSelector/index.tsx

```diff
@@ -228,6 +228,30 @@ export const HourSelector: FC<HourSelectorPropType> = ({ activeHourKey }) => {
                 </g>
               )
             })}
+            <g
+              role="button"
```

Comment:
The `button` role should be removed here as it is not an interactive element. Probably even aria-hiding it is a good idea.